### DISCUSSION TITLE
clash-core不更新了，现在是否应该考虑转向到mihomo或者singbox吗？

### DISCUSSION THREAD

**Discussion by sss132919** [vernesong/OpenClash#3754] [Ideas]
clash-core不更新了，现在是否应该考虑转向到mihomo或者singbox吗？

**Comment by ghost** [comment]
mihomo:
![1](https://github.com/vernesong/OpenClash/assets/152784766/77be9253-48bc-4cb1-bb0a-65fa2e6323a2)
R U OK?

singbox是独立开发的，mihomo和clash同源。
所以支持singbox=代码重构，目前你应该也没看到基于clash的工具，在没支持开发者的情况下只换个singbox内核就能正常运行的吧。
当然如有能力欢迎PR。

**Reply by sss132919** [reply]
期待luci-app-opensingbox、luci-app-openmihomo的出现

**Reply by ghost** [reply]
不是，额。。。
也怪我说的不清
meta=M核=mihomo，现在已经是被支持的了，你只要去`插件设置`里勾选`使用meta内核`

然后多嘴一句。截至目前的singbox版本，哪怕机场已经提供了“开封即食”的文件，如果靠自己“手搓”clash的配置文件都做不到，那我还是建议别琢磨它了。除非只有最基础的上网需求，否则该如何让一个原属于直连的网站走代理都做不到。

**Reply by dyss1992** [reply]
@sangyishuje1123 请教一下，OpenClash 从原来的 Clash Meta 切换到现在的 Mihomo 内核需要怎么操作呢？把 Mihomo 改名成 Clash_Meta 吗？

**Reply by ghost** [reply]
> 请教一下，OpenClash 从原来的 Clash Meta 切换到现在的 Mihomo 内核需要怎么操作呢？把 Mihomo 改名成 Clash_Meta 吗？

你好，如果是自己手动通过oc以外的方式上传或下载。是的，你需要。

但是，通过oc自带上传内核功能则不需要，在配置管理上传文件的下拉菜单中。通过oc在线下载更新也是不需要的。

P.S.新春快乐

**Reply by lux5am** [reply]
Dude mihomo is meta. It just the new name. If you enable meta in openclash it will use mihomo. The meta core update in openclash also use mihomo. So just update the meta core through openclash. It will download mihomo core. Mihomo is mihomo. Mihomo also mihomo. are you clear mihomo 🥲

**Reply by ghost** [reply]
> Dude mihomo is meta. It just the new name. If you enable meta in openclash it will use mihomo. The meta core update in openclash also use mihomo. So just update the meta core through openclash. It will download mihomo core. Mihomo is mihomo. Mihomo also mihomo. are you clear mihomo 🥲

Hi, are you replying to me? I’m not sure what you’re trying to say. Could you make it more clear?
Are you against Mihomo changing its name to clash_meta? If so, then you probably don't understand how openclash works.

**Reply by lux5am** [reply]
Mihomo is meta

**Reply by ghost** [reply]
Yes I know that mihomo is meta, and I clearly wrote `meta=M core=mihomo` in my previous reply

**Reply by lux5am** [reply]
Mihomo is meta and mihomo also meta

**Reply by ghost** [reply]
yes, you're right, but I still don't understand what you're talking about.
if you're just trying to tell what meta and mihomo are, then i think we can end at here.
